src/main/java/com/dongtronic/diabot/platforms/discord/listeners/HelpListener.kt

```diff
@@ -1,19 +1,33 @@
 package com.dongtronic.diabot.platforms.discord.listeners
 
-import com.dongtronic.diabot.platforms.discord.commands.DiabotCommand
 import com.dongtronic.diabot.exceptions.NoCommandFoundException
+import com.dongtronic.diabot.platforms.discord.commands.DiabotCommand
 import com.jagrosh.jdautilities.command.Command
 import com.jagrosh.jdautilities.command.CommandEvent
 import net.dv8tion.jda.api.EmbedBuilder
 import net.dv8tion.jda.api.Permission
 import net.dv8tion.jda.api.entities.ChannelType
+import net.dv8tion.jda.api.exceptions.ErrorResponseException
 import net.dv8tion.jda.api.exceptions.InsufficientPermissionException
+import net.dv8tion.jda.api.requests.ErrorResponse
 import java.awt.Color
-import java.security.Permissions
 import java.util.*
 import java.util.function.Consumer
 
 class HelpListener : Consumer<CommandEvent> {
+    /**
+     * Executed when the attempt to send a DM to a user fails
+     */
+    private fun sendingError(exc: Throwable, event: CommandEvent) {
+        if (exc is ErrorResponseException
+                && exc.errorResponse != ErrorResponse.CANNOT_SEND_TO_USER) {
+            // Print stack trace if the error code was not related to DMs being blocked
+            exc.printStackTrace()
```

Comment:
add a logger object to this class (search for `LoggerFactory` in the project for an example) and use `logger.warn` with a meaningful error message too.